worker/dbaccessor/package_test.go

```diff
@@ -11,7 +11,8 @@ import (
 	gc "gopkg.in/check.v1"
 )
 
-//go:generate go run github.com/golang/mock/mockgen -package dbaccessor -destination package_mock_test.go github.com/juju/juju/worker/dbaccessor Logger,DBApp,OptionFactory
+//go:generate go run github.com/golang/mock/mockgen -source worker.go -package dbaccessor -destination package_mock_test.go github.com/juju/juju/worker/dbaccessor Logger,DBApp,OptionFactory
```

Comment:
Notice the `-source` this is a work around to ensure that we don't follow alias types through to dqlite, which causes it to blow up on a mac.